src/parse_utils.rs

```diff
@@ -2,7 +2,7 @@ use nom::{
     bytes::complete::tag,
     character::complete::{digit1, space0, space1},
     combinator::{map, map_res, opt, recognize},
-    multi::separated_nonempty_list,
+    multi::separated_list0,
```

Comment:
Thanks for the PR!

This looks like a change in behaviour - wouldn’t we want separated_list1 here?

Reply:
You are right. Fixed!